src/qmeta/QMetaMysql.cc

```diff
@@ -287,6 +287,8 @@ QueryId QMetaMysql::registerQuery(QInfo const& qInfo, TableNames const& tables)
     query += msgTableName;
     query += ", ";
     query += resultLocation;
+    query += ", ";
+    query += boost::lexical_cast<string>(qInfo.chunkCount());
```

Comment:
No need for this in modern C++11 (or newer). There is the standard library function:
```
std::to_string(qInfo.chunkCount())
```
I suggest refactoring other uses of `boost::lexical_cast<string>()` in this code where it makes sense to do.

Reply:
This was more of a "when in rome" type thing. Using to_string is fine with me.